racadm.py

```diff
@@ -133,71 +137,74 @@ def logout(self):
         Close the session so we are not stealling spaces
         '''
         #this dosn't work because 'Closing the executing session is not permitted.'
```

Comment:
afraid not i think i generally do a `racreset` once im done.  Also keep in mind if you are using this script that the syntax i use is being deprecated so may not work in future.  unfortunatly i havn't had time to rewrite for the new interface yet.